content/2.application.md

```diff
@@ -45,11 +45,11 @@ The Payments CorDapp is a custom CorDapp developed by the Participant or the BNO
 
 ### Gateway node
 
-The Gateway node is a node in the Corda business network where the Gevamu Payments Gateway is installed as a CorDapp.
+The Gateway node is a node in the Corda business network where the Gevamu Payments Gateway is installed.
 Each Gevamu Payments Gateway is custom-written to work with a specific PSP and every payment initiation request requires that the gateway be specified.
 Therefore, with minimal changes, the App using the Payments SDK can connect to different PSPs.
 The Gateway node communicates on-chain with the Participant’s CorDapp based on the Gevamu Payments SDK via a proprietary API; the connection with the PSP is off-chain.
-
+The payment gateway transforms the payment instruction into a format required by the PSP.
```

Comment:
Gateway does not transform instruction. Gateway just checks if instruction is valid and resends it to PSP.
```suggestion
```